sandy_sort_lists.py

```diff
@@ -0,0 +1,94 @@
+import config
+import requests
+import json
+
+
+COLOR_KEYS = {
+	"green": 1,
+	"yellow": 2,
+	"orange": 3,
+	"red": 4,
+	"purple": 5,
+	"blue": 6,
+	"sky": 7,
+	"lime": 8,
+	"pink": 9,
+	"black": 10
+}
+BOARD_IDS_URL = "https://api.trello.com/1/members/{username}/boards"
+GET_LISTS_URL = "https://api.trello.com/1/boards/{board_id}/lists"
+GET_CARDS_URL = "https://api.trello.com/1/lists/{list_id}/cards"
+PUT_CARDS_URL = "https://api.trello.com/1/cards/{card_id}/pos"
+BASIC_PARAMS = {
+        "key": config.TRELLO_APP_KEY,
+        "token": config.TRELLO_APP_TOKEN
+        }
+
+
+def get_key_value(card):
+	try:
+		color = COLOR_KEYS[card["labels"][0]["color"]]
+	except IndexError, e:
+		color = 0
+	return color
+
+
+def get_lists_from_board(board_id):
+	resp = requests.get(GET_LISTS_URL.format(board_id=board_id), params=BASIC_PARAMS)
```

Comment:
To illustrate my point in Trello Python API bindings, this line could be replaced with, for example, something like:

`TrelloAPI.get_lists(board_id, board_id)`, and it takes care of the magic internally :). Of course, you initialize `TrelloAPI` once with the credentials upfrnot